src/components/tl/ucp/tl_ucp_coll.h

```diff
@@ -307,5 +307,20 @@ ucc_status_t ucc_tl_ucp_alg_id_to_init(int alg_id, const char *alg_id_str,
                                        ucc_memory_type_t        mem_type,
                                        ucc_base_coll_init_fn_t *init);
 
+static inline unsigned ucc_tl_ucp_get_sra_kn_radix(ucc_tl_ucp_team_t *team,
```

Comment:
add similar function for allreduc_kn_radix (not sra) so can be used for full allreduce tuning